suzieq/poller/controller/credential_loader/static.py

```diff
@@ -42,7 +42,7 @@ def init(self, init_data: dict):
         if init_data.get('password'):
             password = init_data['password']
             if password.startswith('env:'):
```

Comment:
What about having a function in utils doing this work? So that whenever we need to get some confidential data we can call it without repeating the same code. i.e.
```
password = get_pwd(init_data['password'])
```